common/src/device/transport/mqtt/ClientWebSocketChannelConfig.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client.Transport.Mqtt
+{
+    using System;
+    using System.Diagnostics.Contracts;
+    using DotNetty.Buffers;
+    using DotNetty.Transport.Channels;
+
+    internal class ClientWebSocketChannelConfig : IChannelConfiguration
```

Comment:
can't you inherit from https://github.com/Azure/DotNetty/blob/dev/src/DotNetty.Transport/Channels/DefaultChannelConfiguration.cs?

Reply:
Will do.

Reply:
I removed this class completely and used DefaultChannelConfiguration instead.